build.sbt

```diff
@@ -2036,6 +2057,8 @@ lazy val designer = (project in file("designer/server"))
         "io.circe"                      %% "circe-yaml"                      % circeYamlV           % Test,
         "com.github.scopt"              %% "scopt"                           % "4.1.0"              % Test,
         "org.questdb"                    % "questdb"                         % "7.4.2",
+        "org.apache.kafka"               % "kafka-clients"                   % kafkaV,
```

Comment:
What's going on here?

Reply:
TBH this dependency should be added at the component definition level (in the module when the kafka-related component/s is/are added). For some reason, there is no such dependency in the lite kafka component module (or one of the lite-related kafka modules). It's hard for me to unravel the relationships between these modules, so I decided to put the dependency here as well. 

<img width="1569" alt="Screenshot 2024-12-16 at 23 07 20" src="https://github.com/user-attachments/assets/a391157b-fbb1-4a1e-93e4-083ae85d3489" />